app/controllers/vouchers_controller.rb

```diff
@@ -3,13 +3,16 @@ class VouchersController < ApplicationController
   load_and_authorize_resource
 
   FIELD_FILTER_WHITELIST = [
-    
+    'items.product_id'
```

Comment:
as of now, we can only filter the voucher on the product there are related to

but everything is possible, just tell me the filter we want and it will be super easy to set up :)